### ISSUE TITLE
Is this adapter still maintained ? - PLEASE RESPOND UNTIL 30.6.2024

### ISSUE THREAD

**Issue by mcm1957** [smaragdschlange/ioBroker.nuki#77]
Is this adapter still maintained?

This adapter does not work with js.controller 6 (for now). Please let us know if this adapter is still maintained and will be fixed in the near future.

**If there is no reaction until 30.6.2024 this adapter could be deprected and removed from repositories or eventually modes to iobroker-community-area.**

UPDATE:
@smaragdschlange 

**As a fix is available for js-controlelr 6 I've shortend the response time. Please respond ASAP if possible. If no response is received we will move the adapetr to the community area.**

**Comment by simatec** [comment]
I have made a PR on the known problems... Let's see if the developer releases it...
https://github.com/smaragdschlange/ioBroker.nuki/pull/78

**Comment by smaragdschlange** [comment]
Hi,
thanks @simatec and @mcm1957, I really appreciate the support.
I have merged the PR and I will go on and release it. 
Sorry if my response time is quite poor, but I have a lot on my plate right now...
I actually do use this adapter in my home-setup and I don't experienced any issues so far...it just works as expected. But now I just realized, I'm still on jS-controller 5.0.19.

Thanks again,
Sven
